ImagePickerSheetController/ImagePickerSheetController/ImagePickerSheetController.swift

```diff
@@ -31,13 +31,19 @@ public enum ImagePickerMediaType {
     
 }
 
+fileprivate let imageManager = PHCachingImageManager()
```

Comment:
@lbrndnr This is the crux of the problem/bug-fix.  When this object is deallocated, if permission to photos has not been given, apple issues an exception of "permission not given".  Thanks apple!